templates/index.md.tmpl

```diff
@@ -17,6 +17,8 @@ You must install [1Password CLI](https://developer.1password.com/docs/cli) on th
 
 ## Use with 1Password CLI
 
+Minimum 1Password CLI version `v2.23.0` is required.
```

Comment:
`[non-blocking]`

Should this be highlighted more with emojis or using bold? I worry that it may be easily missed, and it's an important thing to call out.

Reply:
added emoji